include/oneapi/mkl/blas/detail/cublas/blas_ct.hpp

```diff
@@ -47,7 +48,7 @@ void syr2<library::cublas, backend::nvidiagpu>(cl::sycl::queue &queue, uplo uppe
                                                cl::sycl::buffer<float, 1> &y, std::int64_t incy,
                                                cl::sycl::buffer<float, 1> &a, std::int64_t lda) {
     syr2_precondition(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
-    onemkl::cublas::syr2(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
+    oneapi::mkl::cublas::syr2(queue, upper_lower, n, alpha, x, incx, y, incy, a, lda);
```

Comment:
Not related to this MR, but in general, maybe cublas should not be inside mkl namespace, especially now that we have a new namespace called oneapi.

Reply:
Let's bring this up as a proposal later.